ios/browser/api/certificate/models/brave_certificate_rdns_sequence.mm

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/browser/api/certificate/models/brave_certificate_rdns_sequence.h"
+#import "brave/ios/browser/api/certificate/utils/brave_certificate_utils.h"
+#import "brave/ios/browser/api/certificate/utils/brave_certificate_x509_utils.h"
+#import "brave/ios/browser/api/common/conversions.h"
+
+// base
+#include "base/strings/sys_string_conversions.h"
+
+// net
+#include "net/cert/x509_cert_types.h"
+#include "net/der/input.h"
```

Comment:
These should all just be in a single list, no separation (and you can use `#include`)